src/pages/Environments.js

```diff
@@ -116,8 +120,165 @@ const DeleteAppModal = ({ app: { googleProject, appName, diskName, appType }, on
   ])
 }
 
+const MigratePersistentDisksBanner = ({ count }) => {
+  const deadline = new Date('01 January 2023 00:00 UTC')
+  return count > 0 && div({
+    style: {
+      position: 'absolute', top: topBarHeight, left: '50%', transform: 'translate(-50%, -50%)',
+      backgroundColor: colors.warning(0.15),
+      border: '2px solid', borderRadius: '12px', borderColor: colors.warning(),
+      zIndex: 8000 // Must be under 9000 to not draw over notifications
+    }
```

Comment:
Does the banner really need to overlap with the top bar? We could get rid of all of this position, transform, and zIndex stuff if it was across the top of the main content area.

Reply:
According to the UX mock-ups in the ticket, the banner does. I'm just the messenger haha

Reply:
On narrower windows (Terra UI supports down to 1200px), the banner overlaps the title in the top bar.

<img width="1187" alt="Screen Shot 2022-09-08 at 11 21 38 AM" src="https://user-images.githubusercontent.com/1156625/189161524-97d949f6-8f62-4852-84ff-7a364589e140.png">

Reply:
Yeah - that's what the mockup does

Reply:
https://broadworkbench.atlassian.net/jira/software/c/projects/PPWM/boards/150?modal=detail&selectedIssue=PPWM-76